internal/storage/bucket_handle_test.go

```diff
@@ -158,9 +158,29 @@ func (t *BucketHandleTest) TestDeleteObjectMethodWithMissingObject() {
 	error := t.bucketHandle.DeleteObject(context.Background(),
 		&gcs.DeleteObjectRequest{
 			Name:                       missingObjectName,
-			Generation:                 0,
+			Generation:                 TestObjectGeneration,
 			MetaGenerationPrecondition: nil,
 		})
 
 	AssertEq("storage: object doesn't exist", error.Error())
 }
+
+func (t *BucketHandleTest) TestStatObjectMethodWithValidObject() {
+	_, error := t.bucketHandle.StatObject(context.Background(),
+		&gcs.StatObjectRequest{
+			Name:              TestObjectName,
+			ForceFetchFromGcs: true,
```

Comment:
this field is not used in your code. you can ignore setting it.

Reply:
fixed